target/linux/pistachio/creator-platform-default-cascoda.config

```diff
@@ -11,3 +11,6 @@ CONFIG_VERSION_MANUFACTURER="Imagination Technologies"
 CONFIG_VERSION_MANUFACTURER_URL="www.imgtec.com"
 CONFIG_VERSION_PRODUCT="Creator Ci40(Marduk)"
 CONFIG_VERSION_HWREV="Ci40 with CA8210"
+CONFIG_DEVEL=y
```

Comment:
We should not be adding hacks into this config file as the users won't get if they build it. The proper way to do this is to add 'fpu' as a feature in target/linux/pistachio/Makefile